src/js/GaModule.js

```diff
@@ -221,4 +221,8 @@ goog.require('ga_waitcursor_service');
     whitelist = whitelist.concat(gaGlobalOptions.whitelist);
     $sceDelegateProvider.resourceUrlWhitelist(whitelist);
   });
+
+  module.config(function($compileProvider) {
+    $compileProvider.aHrefSanitizationWhitelist(/^\s*(https?|whatsapp|file|s?ftp|blob|mailto|chrome-extension):/)
```

Comment:
We already have a whitelist [there](https://github.com/geoadmin/mf-geoadmin3/blob/master/src/index.mako.html#L785) . you can add the wahtsapp url there.  I think it's safer than an obscure untested regex.

Reply:
no but it's a whatsapp protocol. this won't work

Reply:
There is a default white list of href protocols in angularjs, anything that is not whitelisted here will be prefixed with `unsafe:`

Reply:
ok I didn't see it was a protocolo
But it could be great if the regex comes from the `gaGlobaOptions` object. And a test woul be nice.

Reply:
agree